virtool/auth/utils.py

```diff
@@ -0,0 +1,93 @@
+from logging import getLogger
+
+import openfga_sdk
+from aiohttp import ClientConnectorError
+from openfga_sdk import (
+    CreateStoreRequest,
+    WriteAuthorizationModelRequest,
+    TypeDefinition,
+    Userset,
+)
+from openfga_sdk.api import open_fga_api
+
+logger = getLogger("OpenFGA")
+
+
+async def connect_openfga():
+    configuration = openfga_sdk.Configuration(
+        api_scheme="http", api_host="localhost:8080"
+    )
+
+    logger.info("Connecting to OpenFGA")
+
+    try:
+        api_client = openfga_sdk.ApiClient(configuration)
+        api_instance = open_fga_api.OpenFgaApi(api_client)
+
+        await create_store(api_instance, configuration)
+        await write_auth_model(api_instance)
+
+    except ClientConnectorError:
+        logger.fatal("Could not connect")
```

Comment:
This is missing an important call. See other uses of `logger.fatal` in Virtool for guidance.